examples/streaming/README.md

```diff
@@ -65,9 +65,28 @@ check check check
 ]
 ```
 
+### `subscribe` Endpoint
+
+This endpoint illustrates **streaming result**. The client sends a
+payload defined in `Payload` DSL containing the JWT token. Once auth succeeds,
+the server establishes a stream through which it sends events to the client
+when a new message is added. The subscription ends when the client goes away
+or the server shuts down.
+
+```
+$ ./chatter_cli chatter subscribe --token $JWT_TOKEN
+# Now send messages to server in a separate `chatter_cli` process by calling
+# the `summary`, `listener`, or `echoer` endpoints.
+{
+    "Message": "hello",
+    "Action": "added",
+    "AddedAt": "2018-08-14T12:32:30-07:00"
+}
+```
+
 ### `history` Endpoint
 
-This endpoint illustrates **streaming result**. The client sends a payload
+This endpoint illustrates **streaming result** with views. The client sends a
 payload defined in `Payload` DSL containing the JWT token and an optional
 "view" parameter. Once auth succeeds, the server streams all the
 messages sent by the client rendered using the optional "view" parameter.
```

Comment:
Is it sent by the client or to the client? Should that be:
The server sends results to the client using the stream. Each result struct is first projected to the view matching the the "view" parameter.